app/controllers/miq_ae_class_controller.rb

```diff
@@ -1880,7 +1880,7 @@ def deleteinstances
 
     process_aeinstances(aeinstances, "destroy") unless aeinstances.empty?
     add_flash(_("The selected %{record} were deleted") %
-      {:record => ui_lookup(:models => "MiqAeInstances")}) if @flash_array.nil?
+      {:record => ui_lookup(:models => "MiqAeInstance")}) if @flash_array.nil?
```

Comment:
Originally, the line read: `models => "MiqAeInstances"` which was incorrect, because there is no such
model name: `MiqAeInstances`. There is only a model name `MiqAeInstance`.

If you say `:model => ...`, `ui_lookup` will return (translated) singular version of the model name.

If you say `:models => ...`, `ui_lookup` will return (translated) plural version of the model name you want.

So in this case, yes, using `:models` is correct, because I believe the original intention behind using 
`MiqAeInstances` rather than `MiqAeInstance` was to get the plural form of this model name.